### ISSUE TITLE
Sorting pages

### ISSUE THREAD

**Issue by lahoje** [BookStackApp/BookStack#5074]
### Describe the Bug

After draging and dropping the pages in disired order and saving it, it doesn't sort as I saved it?

### Steps to Reproduce

1. Goto sorting a Book either under Book (Sort Book) or Chapter (Sort).
2. Sort pages under a chapter of you choice.
3. Save.
4. Review the change - it still is sorted as before sorting i as described above.

### Expected Behaviour

The pages should be sorted as you saved them.

### Screenshots or Additional Context
**BEFORE SAVE AFTER SORTING**
![Sort - after](https://github.com/BookStackApp/BookStack/assets/32420933/214c084a-88b4-467e-bdc4-a87a384552ec)

**AFTER SAVE**
![Sort - before](https://github.com/BookStackApp/BookStack/assets/32420933/6209726e-9c48-4c2f-a877-b3fd5af350fa)

### Browser Details

MS Edge Version 126.0.2592.61 (Officiell version) (arm64); Chrome Version 126.0.6478.62 (Officiell version) (arm64)

### Exact BookStack Version

BookStack v24.05.1

**Comment by kilian-goetz** [comment]
Greetings @lahoje,

I hope you're well. Could you please provide more details, such as the Laravel logs? They are often located in /app/www/storage/logs.

Kind regards, Kilian GOËTZ.

**Comment by ssddanbrown** [comment]
@Beladric I don't think the logs would be too relevant here since there doesn't seem to be a specific error thrown.

@lahoje Can you please provide some detail in regard to how you're hosting BookStack? (Install method, Hosting method, host OS, service used etc...)

**Comment by lahoje** [comment]
@ssddanbrown Hello and thanks for your reply. I have to ask my ISP. I've attached the laravel log in case it could point at some direction in the meanwhile.
[laravel.log](https://github.com/user-attachments/files/15910216/laravel.log)

**Comment by lahoje** [comment]
Hi again, here's requested information installation method etc:

Install method: Manual installation through Git clone
Hosting method: Litespeed webserver, CloudLinux OS, PHP 8.2

**Comment by ssddanbrown** [comment]
@lahoje Is this some kind of hosting provider? If so, Do you have access to choose/change/select which PHP extensions are active or in use?

**Comment by lahoje** [comment]
@ssddanbrown Yes it's a hosting provider and I can see how I can manage php extensions using their management console.

**Comment by lahoje** [comment]
Hi again. we found the problem in the issue report #4700 regarding nd_pdo issue. I've submitted a PR to clarify the option for nd_pdo_mysql.

**Comment by ssddanbrown** [comment]
BookStack is now [managed on Codeberg](https://codeberg.org/bookstack/bookstack).

As part of this change, we're closing all existing GitHub issues and pull requests.
You'll instead find this issue on Codeberg here: 
https://codeberg.org/bookstack/bookstack/issues/5074
  
Any further discussion for this will now be done on Codeberg.
Therefore this GitHub thread will be locked.
